src/plugins/jni/README.md

```diff
@@ -47,6 +48,25 @@ and run:
 
     sudo ldconfig
 
+### Java prerequisites on macOS ###
+
+macOS includes an old apple specific version of java, based on 1.6. However, for the jni plugin version 1.8 of Java is required, so either the openjdk or the oracle jdk has to be installed.
```

Comment:
Please use linebreaks.

Did you try if java 1.6 works? As far as I can remember the constant `JNI_VERSION_1_8` is the only bit actually requiring 1.8.